src/config/main-navigation-options.ts

```diff
@@ -11,8 +12,18 @@ export default function (req: any, res: Response, next: NextFunction): void {
     })
 
     const userProfile = req.appSession?.userProfile ?? {}
-    const profileUrl = userProfile.profile ?? null
-    const groupUrl = userProfile['bas:groupProfile'] ?? null
+    let profileUrl = userProfile.profile ?? null
+    let groupUrl = userProfile['bas:groupProfile'] ?? null
+    const baseUrl: string = req.app.get('baseURL') ?? ''
+
+    if (profileUrl != null) {
+      profileUrl = profileUrl + '?redirect_uri=' + encodeURIComponent(baseUrl) + '/auth/profile-redirect'
```

Comment:
Won't really matter in this case, but it's probably good practice to encode the whole parameter, something like:
```suggestion
      profileUrl = profileUrl + '?redirect_uri=' + encodeURIComponent(baseUrl + '/auth/profile-redirect')
```

Reply:
encoded the whole parameter